modules/apps/frontend-js/frontend-js-react-web/webpack.config.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const path = require('path');
+const { container: { ModuleFederationPlugin } } = require('webpack');
+
+module.exports = {
+	context: path.resolve(__dirname),
+	devtool: 'source-map',
+	entry: './null.js',
+	mode: 'development',
+	module: {
+		rules: [
+			{
+				exclude: /node_modules/,
+				test: /\.js$/,
+				use: {
+					loader: 'babel-loader',
+					options: {
+						presets: ['@babel/preset-react'],
+						plugins: [
+							'@babel/plugin-proposal-class-properties'
+						]
+					},
+				},
+			},
+			{
+				exclude: /node_modules/,
+				test: /\.scss$/,
+				use: [
+					'style-loader',
+					'css-loader',
+					'sass-loader',
+				],
+			},
+		],
+	},
+	output: {
```

Comment:
Feels like most of this should be autogenerated in `liferay-npm-scripts`

Reply:
I would say everything but the entry points :+1: .